ts/screens/wallet/payment/ScanQrCodeScreen.tsx

```diff
@@ -292,7 +292,9 @@ class ScanQrCodeScreen extends React.Component<Props, State> {
 const mapDispatchToProps = (dispatch: Dispatch) => ({
   navigateToWalletHome: () => dispatch(navigateToWalletHome()),
   navigateToPaymentManualDataInsertion: () =>
-    dispatch(navigateToPaymentManualDataInsertion()),
+    dispatch(
+      navigateToPaymentManualDataInsertion({ isInvalidAmount: undefined })
```

Comment:
Are you sure `isInvalidAmount: undefined` is needed?

Reply:
@fpersico I have made "params" optional in navigation.ts and now it is no longer necessary. (b28d7f1)